zookeeper-api/src/main/java/org/apache/helix/zookeeper/zkclient/callback/ZkAsyncCallbacks.java

```diff
@@ -170,7 +170,17 @@ public void callback(int rc, String path, Object ctx) {
       }
 
       if (ctx != null && ctx instanceof ZkAsyncCallMonitorContext) {
```

Comment:
You still need to change the asynGet method to catch exception to make sure the ctx is an instance of `ZkAsyncCallMonitor` instead of `ZkAsyncRetryCallContext` right?

Reply:
Yes, I think so. I'm discussing that with @pkuwm.